ctapipe_io_lst/calibration.py

```diff
@@ -508,12 +532,9 @@ def time_lapse_corr(self, event, tel_id):
 
     def interpolate_spikes(self, event, tel_id):
         """
-        Interpolates spike A & B.
-        Fill the R1 container.
-        Parameters
-        ----------
-        event : `ctapipe` event-container
-        tel_id : id of the telescope
+        Interpolate spikes at known positions from their neighboring values
+
+        Mutates the R1 waveform.
```

Comment:
no `Parameters` or `Returns` given

Reply:
It does not have a return value and the parameters are `event` and `tel_id`, do these really need explanations?

Reply:
ok